### PR TITLE
Salesforce lwc

### PR DESCRIPTION
**Pull request by matt-degraffenreid**
@jblakey-google's implementation for integrating the agent assist ui module with salesfoce

### CODE REVIEW

**Review by jblakey-google — APPROVED** [review]
LGTM. The diff appears to be identical with #20 that @nategd-google approved.
Ah, but you need a review from someone with write access.

**Review by nategd-google — APPROVED** [review]

**Review by nategd-google — APPROVED** [review]

**Pull request merged**
